test/e2e/externalservices_universal_test.go

```diff
@@ -64,6 +65,8 @@ networking:
   address: %s
   tls:
     enabled: %s
+    ca_cert:
```

Comment:
does it work? I was pretty sure that you can only use camel case in our YAMLs - `caCert`

Reply:
Actualy both `ca_cert` and `caCert` work when tested. Will stay with the latter as suggested. Thanks.